packages/react-component-library/src/components/DatePicker/DatePicker.tsx

```diff
@@ -17,15 +22,15 @@ export interface StateObject {
   to?: Date
 }
 
-export interface DatePickerProps extends ComponentWithClass {
+export interface DatePickerProps extends ComponentWithClass, DayPickerProps {
```

Comment:
Looks like extending from `DayPickerProps` is bringing in a lot of stuff which seems to be mostly discarded – perhaps it could use a `Pick<>` or alternatively `disabledDays: DayPickerProps['disabledDays']`?